CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+## Welcome
+
+If you are a student at North Seattle College we welcome you to contribute to our project! Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**. If you are nervous that is okay. The community will help guide you. Mistakes will be made, we will all learn together, and everything will be fine!
+
+Not all contributions are in the form of code. In fact, a great way to learn about the project is dig into the documentation and poke around the website. Then do any of the following:
+
+- Add new or editing existing documentation
+- Submit a bug report
+- Submit a feature request
+- Ask a question
+
+## How to Contribute
+Navigate to the `Issues` tab and use the filters to see open issues. Take note of the `good first issue` and `help wanted` labels.
+
+#### Submit a Bug
+- Navigate to `Issues` and click the `New Issue` button
+- Select `Bug Report`
+
+#### Submit a Feature Request
+- Navigate to `Issues` and click the `New Issue` button
+- Select `Feature Request`
```

Comment:
This seems to jump from PR to Issues to PR. Maybe move the issues above How to Contribute.

Reply:
Consolidated and moved to readme. Trying a draft where the setup is all in CONTRIBUTING.md and the Issues which can be logged by anyone with a github account are in the README.md. Let me know if you like it. I think it balances the length and content of both documents fairly well.